src/site/markdown/index.md

```diff
@@ -0,0 +1,173 @@
+<!---
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+
+****************
+Introduction
+****************
+Stephen Connolly
+****************
+2009-04-21
+****************
```

Comment:
Meta date can be put in markdown ... 
it is some of hidden doxia feature https://issues.apache.org/jira/browse/DOXIA-617 😄

Reply:
Also:

> So, no, the YAML header parsing is not enabled. The trickiest part will be the backward compatibility with the "old" syntax (without the --- dashes). If we could manage to remove the code that currently handles the header section, that would be awesome. We would detect the old syntax header, and we would surround it with dashes so that it's handled automatically with Flexmark's YAML parser.

Reply:
Gotcha, no dashes 😆